modules/installation-configuration-parameters.adoc

```diff
@@ -313,6 +319,15 @@ ifdef::osp[]
    }
 }
 ----
+
+|`platform.openstack.machinesSubnet`
+|The UUID of a {rh-openstack} subnet that the cluster's nodes use.
```

Comment:
It might be helpful to also specify that the api and ingress ports will be created on this network.